anevar/nevar_DZ_2/DZ_2_1_1.html

```diff
@@ -12,11 +12,11 @@
 				<th>Город</th><th>Утро</th><th>День</th><th>Вечер</th><th>Ночь</th>
 			 </tr>
 			 <tr>
-				<td><b>Минск</b></td><td>-5&deg;</td><td>+5&deg;</td><td rowspan="2">+2&deg;</td><td>-10&deg;</td>
+				<th><b>Минск</b></th><td>-5&deg;</td><td>+5&deg;</td><td rowspan="2">+2&deg;</td><td>-10&deg;</td>
```

Comment:
You did not remove tag b. th tag applies  bold style for contend. Is it not necessary to duplicate styles